source/fuzz/transformation_propagate_instruction_up.cpp

```diff
@@ -0,0 +1,397 @@
+// Copyright (c) 2020 Vasyl Teliman
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "source/fuzz/transformation_propagate_instruction_up.h"
+
+#include "source/fuzz/fuzzer_util.h"
+#include "source/fuzz/instruction_descriptor.h"
+
+namespace spvtools {
+namespace fuzz {
+namespace {
+
+std::unordered_map<uint32_t, std::unordered_map<uint32_t, uint32_t>>
+ComputeMappingFromOpPhiToResultId(opt::IRContext* ir_context,
+                                  opt::Instruction* inst) {
```

Comment:
Consider commenting the body of this method a little.

Reply:
Fixed in 4e30a81.